src/Services/categoriesServices.js

```diff
@@ -1,14 +1,67 @@
-import axios from "axios";
+import axios from 'axios'
 
-const getCategory = async () => {
+const baseURL = 'http://ongapi.alkemy.org/public/api'
+const headers = {
+  'Content-Type': 'application/json',
+  Accept: 'application/json',
+}
+
+export const getCategories = async () => {
   try {
     const response = await axios.get(
-      "http://ongapi.alkemy.org/public/api/categories"
-    );
-    return response.data.data;
+      'http://ongapi.alkemy.org/public/api/categories'
+    )
+    return response.data.data
```

Comment:
use the same logic for all the functions. If you use response.data.data, then return directly that on getCategory, and the same on createCategory.
If you use
``` 
    const { data } = axios.post(`${URL}/categories`, categoryData, {
      headers: headers,
    })
    return data
```
then, use that on all the functions